nx-dev/nx-dev/redirect-rules.js

```diff
@@ -418,7 +418,9 @@ const nxCloudUrls = {
   '/nx-cloud/set-up/github':
     '/nx-cloud/recipes/source-control-integration/github',
   '/nx-cloud/set-up/bitbucket-cloud':
-    '/nx-cloud/recipes/source-control-integration/bitbucket-cloud',
+    '/nx-cloud/recipes/source-control-integration/bitbucket',
+  '/nx-cloud/recipes/source-control-integration/bitbucket-cloud':
+    '/nx-cloud/recipes/source-control-integration/bitbucket',
```

Comment:
is it possible to rewrite these to map to `/ci/` instead of `/nx-cloud/` to reduce the redirects?

Reply:
👍 makes sense, will do